scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -417,7 +417,9 @@
                                  :pod-supports-cook-sidecar? false
                                  :task-id (str api/cook-synthetic-pod-name-prefix "-" pool-name "-" uuid)
                                  :task-request {:scalar-requests (walk/stringify-keys pool-specific-resources)
-                                                :job {:job/pool {:pool/name synthetic-task-pool-name}}}})))
+                                                :job {:job/pool {:pool/name synthetic-task-pool-name}
+                                                      :job/environment environment}
+                                                :resources pool-specific-resources}})))
```

Comment:
Might be nice to add a comment here explaining why we need `:resources` (which I think is only for gpus?).

Reply:
Sure, added